lib/msf/core/web_services/json_rpc_app.rb

```diff
@@ -27,7 +27,7 @@ class JsonRpcApp < Sinatra::Base
       set :show_exceptions, false
 
       set :sessions, {key: 'msf-ws.session', expire_after: 300}
-      set :session_secret, ENV.fetch('MSF_WS_SESSION_SECRET', SecureRandom.hex(16))
+      set :session_secret, ENV.fetch('MSF_WS_SESSION_SECRET', SecureRandom.hex(32))
```

Comment:
Not a blocker; this would technically be a breaking change for our API - but I can't imagine anyone actually setting this value - and there's no results on Google/Github either so looks good to me

Reply:
We could consider calling this a version bump, I suspect you are correct this is not being used externally.